File: .NET/Demo_Common/Entity/eStationMessage.cs
using Demo_Common.Enum;
using MessagePack;

namespace Demo_Common.Entity
{
    [MessagePackObject]
    public class eStationMessage
    {
        /// <summary>
        /// Message code
        /// </summary>
        [Key(0)]
        public MessageCode Code { get; set; } = MessageCode.OK;
    }
}

File: Python/Entities/eStationMessage.py
import msgpack
from dataclasses import dataclass
from Enums.messageCode import MessageCodes

@dataclass
class eStationMessage:
    """
    eStation message
    """
    Code: MessageCodes = MessageCodes.Ok

    def to_msgpack(self) -> bytes:
        return msgpack.packb([self.Code.value])

    @staticmethod
    def from_msgpack(packed: bytes):
        data = msgpack.unpackb(packed)
        return eStationMessage(Code=MessageCodes(data[0]))